Tabloid/client/src/components/TagView.js

```diff
@@ -22,9 +22,13 @@ export const Tags=()=>{
         <>
         <div>
             <h2>Tags:</h2>
-           {tags.map(t=>
-            <h4 key={t.id}> {t.name}</h4> 
-            )} 
+           {tags?.map(t=>
+            <><h4 key={t.id}> {t.name}</h4><Link to={`/tag/delete/${t.id}`}> Delete</Link></>
+
+            
+            )
+            
+            }  
 
           <Link to="/tag/new">Create a tag</Link>
```

Comment:
The last delete link is right next to create a tag. Is there a way to put create a tag underneath it?

Reply:
@SchmidtFred added a div to the create new tag!